qemu/tests/qsd_block_mirror.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from avocado.utils import memory
+
+from provider import backup_utils
+from provider.blockdev_mirror_wait import BlockdevMirrorWaitTest
+from provider.qsd import QsdDaemonDev
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class QSDMirrorTest(BlockdevMirrorWaitTest):
+
+    def __init__(self, test, params, env):
+        super(QSDMirrorTest, self).__init__(test, params, env)
+        self._source_nodes = ["fmt_%s" % src for src in self._source_images]
```

Comment:
Looks like we can remove this function if we set node-name in qsd_image_format in cfg, but it's OK for me here

Reply:
Keep it as it is now, thanks @zhencliu